5740-2020-Robot/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -33,10 +32,10 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-
-  private final Drivetrain m_drivetrain = new Drivetrain(); // Robot Drivetrain
-  private final DriveSlowly m_autoCommand = new DriveSlowly(m_drivetrain);
-  //private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
+  private ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
+  private Drivetrain m_drivetrain = new Drivetrain(); // Robot Drivetrain
+  //rivate final Command m_autoCommand;
+  private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
```

Comment:
same as above replace the "example command" with something we will continue to use.